README.md

```diff
@@ -45,7 +45,7 @@ Commands:
   history   List your recently played tracks.
   next      Play the next track in the queue.
   pause     Pause playback.
-  play      Resume playback or search for a track/album/playlist to play.
+  play      Resume playback, play a Spotify URI or search for a track/album/playlist to play.
```

Comment:
think would look better as:

Resume playback, search for a track/album/playlist, or select a URI to play.

that would probably put them in order of how often they'd be used :)

Reply:
You are right :-)